### DISCUSSION TITLE
Can we set linear gradient color for Path Fill property?

### DISCUSSION THREAD

**Discussion by YogeshwaranCM** [xamarin/Xamarin.Forms#11797] [General]
This is my code. I have set linear gradient bruch inside the path fill. But it is not showing the gradient color. Is there any way to achieve this?

```
<Path
        Aspect="Fill"
        Data="m1.4975,3.03975l345.99999,0l0,116.15857c-191.99999,-107 -172.99999,44.25855 -345.99999,19.11165l0,-135.27022z"
        HorizontalOptions="FillAndExpand">
        <Path.Fill>
            <LinearGradientBrush StartPoint="0,0" EndPoint="1,0">
                <GradientStop Offset="1.0" Color="#E91E63" />
                <GradientStop Offset="0.1" Color="#6A1B9A" />
            </LinearGradientBrush>
        </Path.Fill>
    </Path>
```

**Comment by jsuarezruiz** [comment]
Shapes allows the use of Brushes for filling and for the border.
<img width="343" alt="Captura de pantalla 2020-08-18 a las 17 40 16" src="https://user-images.githubusercontent.com/6755973/90534380-f53e1380-e179-11ea-9522-a2b0659d4a39.png">

If you invert the Offset values, the result is:
<img width="353" alt="Captura de pantalla 2020-08-18 a las 17 39 57" src="https://user-images.githubusercontent.com/6755973/90534390-f7a06d80-e179-11ea-9a89-14707cc07447.png">
If you want to change the orientation of the colors, use the `StartPoint` and `EndPoint` to get the desired direction (vertical, horizontal, diagonal, etc).
